projects/netgrif-components/src/lib/navigation/quick-panel/components/impersonate-quick-panel/impersonate-quick-panel.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component} from '@angular/core';
+import {ImpersonationService, ImpersonationUserSelectService, UserService} from 'netgrif-components-core';
```

Comment:
this inport should be @netgrif/components-core

Reply:
done